config/config.exs

```diff
@@ -45,6 +45,7 @@ config :crawly,
 
   # TODO: this looks outdated
   follow_redirect: true,
+  enable_file_logging: false,
```

Comment:
Can we change it to `log_to_file`? so that it will be similar to the `log_dir` setting?

Reply:
Done